workos/sso.py

```diff
@@ -121,7 +130,7 @@ def list_connections(
         before: Optional[str] = None,
         after: Optional[str] = None,
         order: PaginationOrder = "desc",
-    ) -> SyncOrAsyncListResource: ...
+    ) -> SyncOrAsync[ConnectionsListResource]: ...
```

Comment:
Same here re: `SyncOrAsync` of both the `AsyncWorkOsListResource` and `WorkOsListResource`